Tests/DataPipeline/DataPipelineImplementation+ScreenViewsTest.swift

```diff
@@ -6,25 +6,30 @@ import SwiftUI
 import UIKit
 import XCTest
 
-class DataPipelineImplementationScreenViewsTest: IntegrationTest {
-    private var autoTrackingScreenViews: AutoTrackingScreenViews!
-    private var outputReader: OutputReaderPlugin!
+class DataPipelineScreenViewsTest: IntegrationTest {
+    private var autoTrackingScreenViews: AutoTrackingScreenViews?
+    private var outputReader: OutputReaderPlugin?
```

Comment:
Can we consider suppressing the lint rule for these properties instead? It seems to unnecessarily complicate current code and adding more tests to this file. Given that tests run in a controlled environment and we know the cast always succeeds, I would prefer suppressing lint in cases where following it makes code readability more difficult 🤓

Reply:
I am open to suppressing these, i do think values being nullable can cause test being flaky, and hence i add to ensure checking null everywhere. 

Since i have two votes on going forward with force casting, i am going to go ahead and update it.